indico/web/client/js/jquery/widgets/jinja/person_link_widget.js

```diff
@@ -544,9 +544,11 @@ import {$T} from 'indico/utils/i18n';
       );
     };
 
-    ReactDOM.render(
-      <UserSearchWrapper />,
-      document.getElementById(`principalField-${options.fieldId}`)
-    );
+    if (!options.disableUserSearch) {
```

Comment:
I don't think this is needed anymore - we already have `disabled={options.disableUserSearch}` that disables the button if user search is not allowed.

Reply:
`disabled={options.disableUserSearch}` may be broken. It only disables the button.

Reply:
Yep, that's the intended behavior - I don't think it's wrong to disable the button instead of hiding it...